.github/workflows/maven.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         runs-on: [ ubuntu-latest, macos-latest, windows-latest ]
     name: Jdk ${{ matrix.java-version }}, os ${{ matrix.runs-on }}
     runs-on: ${{ matrix.runs-on }}
+    outputs:
+      output1: ${{ steps.getProjectVersion.outputs.test }}
```

Comment:
Why not actually call it version or something meaningful?

Reply:
still playing to get something working
after that will rename it